src/c++/vdo/tests/asyncLayer.c

```diff
@@ -202,9 +202,9 @@ void initializeAsyncLayer(PhysicalLayer *syncLayer)
 {
   AsyncLayer *asyncLayer;
   VDO_ASSERT_SUCCESS(uds_allocate(1, AsyncLayer, __func__, &asyncLayer));
-  VDO_ASSERT_SUCCESS(vdo_make_int_map(0,
-                                      0,
-                                      &asyncLayer->completionEnqueueHooksMap));
+  VDO_ASSERT_SUCCESS(vdo_int_map_create(0,
+					0,
+					&asyncLayer->completionEnqueueHooksMap));
   VDO_ASSERT_SUCCESS(uds_init_mutex(&asyncLayer->mutex));
```

Comment:
Really tempted to unwrap this line as long as you're touching it.

Reply:
done.